### ISSUE TITLE
Not able to use a SLIDER for cover

### ISSUE THREAD

**Issue by alexbarcelo** [resoai/TileBoard#209]
I may be doing something wrong. But I was trying to have a cover controlled by a `TYPES.SLIDER` and could not get the slider bar to have correct information.

I have:

```
{
  position: [1, 0],
  id: 'cover.cover_living',
  title: 'MyCover',
  type: TYPES.SLIDER,
  unit: '%',
  state: false,
  value: '@attributes.current_position',
  slider: {
    request: {
      type: "call_service",
      domain: "cover",
      service: "set_cover_position",
      field: "position"
    }
  }
},
```

What am I missing? I tried to add a `value` in the top level dictionary, or add a `field` item with no luck.

I believe that [this line](https://github.com/resoai/TileBoard/blob/6119c17ae307c67c1adcc45b1a7f29feed51ee5f/scripts/controllers/main.js#L717) may be the key point, but... I don't fully understand the variables. And I have run out of ideas.

**Comment by alexbarcelo** [comment]
To add a little more info on what I tried:

This shows an arbitrary position:

```
...
slider: {
  value: 35,
  ...
}
...
```

But I tried things like `value: '@attributes.current_position'` and does not work, maybe the context is not valid? I also tried to put full `&cover.cover_living.attributes.current_position'` with no luck, same Angular error:

`Error: "[ngModel:numfmt] http://errors.angularjs.org/1.7.2/ngModel/numfmt?p0=%40attributes.current_position"`

**Comment by alexbarcelo** [comment]
Solved by #210 

Thanks @Evgeny- !

I am not a js guy, and I was fearing being very wrong from the start :)
